src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -318,13 +319,28 @@ private void testEditQuestionAction() throws Exception {
         feedbackEditPage.waitForElementVisibility(visibilityMessage);
 
         feedbackEditPage.verifyHtmlMainContent("/instructorFeedbackQuestionVisibilityPreview.html");
+
+        ______TS("test new question (frame) link copies custom number of recipients option");
+        feedbackEditPage.clickNewQuestionButton();
+        feedbackEditPage.selectNewQuestionType("TEXT");
+        assertTrue(feedbackEditPage.isCustomNumOfRecipientsChecked());
+        feedbackEditPage.clickDiscardChangesLink(-1);
```

Comment:
Looks like -1 has been used consistently throughout to represent the question number of a new question frame (as opposed to an existing question). I think this belongs in another issue altogether? Noticed it in lots of other browser tests that use `InstructorFeedbackEditPage`.